src/Asset/Estate/Estate.model.js

```diff
@@ -63,4 +64,15 @@ export class Estate extends Model {
         WHERE ${BlockchainEventQueries.byArgs('_estateId', estateId)}
           OR (${BlockchainEventQueries.byArgs('_tokenId', estateId)} AND address = ${address})`)
   }
+
+  static async updateEstateDistrictIds() {
+    return this.db.query(
+      SQL`UPDATE ${SQL.raw(this.tableName)} 
+      SET district_id = P.district_id
+        FROM parcels as P 
+        WHERE ${SQL.raw(this.tableName)}.id = P.estate_id AND 
```

Comment:
You could also use an alias here

Reply:
`UPDATE` does not work with alias :( if you do something like 

`UPDATE estates as e SET e.district_id = 1` it fails because e is not a valid column from table estates